tools/release.sh

```diff
@@ -151,6 +142,7 @@ fi
 mkdir kgs
 kgs="kgs/$version"
 pip freeze | tee $kgs
+pip install pytest
```

Comment:
I noticed that with this change `pytest` is no longer pinned. To fix that, how about we include `--dev` in the `poetry export` command and reuse the constraints file here. What do you think?

Reply:
Totally, I do that!